### PR TITLE
Document the workaround for docker for mac and Infinispan

### LINKED ISSUE #148: infinispan-client quickstart can't run using docker - server case
infinispan-client quickstart can't run using docker (both client and server cases)

Instructions in README suggest to run ISPN server in docker or on the main system.
Running server in docker causes stuck client application, I had to `kill -9` it.
I think it's because of the docker nature where the server is on different network than client.

Errors from client when running against ISPN server in docker:
```
2019-04-12 09:08:19,549 INFO  [org.inf.cli.hot.imp.tra.net.ChannelFactory] (HotRod-client-async-pool-0)
   ISPN004014: New server added(172.17.0.3:11222), adding to the pool.
2019-04-12 09:08:19,549 INFO  [org.inf.cli.hot.imp.tra.net.ChannelFactory] (HotRod-client-async-pool-0)
   ISPN004016: Server not in cluster anymore(localhost:11222), removing from the pool.
2019-04-12 09:09:19,548

  WARN  [org.inf.cli.hot.imp.tra.net.ChannelFactory] (HotRod-client-async-pool-1)
  ISPN004015: Failed adding new server 172.17.0.3:11222: io.netty.channel.ConnectTimeoutException:
  connection timed out: /172.17.0.3:11222
```

### PR DESCRIPTION
**Pull request by karesti**
Document a known issue for docker for mac users

Closes #148

### CODE REVIEW

**Comment by karesti** [comment]
ok

**Comment by karesti** [comment]
@rsvoboda done

File: infinispan-client/README.md [hunk 1]
```diff
@@ -4,10 +4,16 @@ This example showcases how to use the Infinispan client with Quarkus.
...
 
-<!-- TODO: https://github.com/quarkusio/quarkus-quickstarts/issues/148
-- Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
--->
-- Download the server (e.g. 9.4 or 10.0) from `http://www.infinispan.org/` and run `./bin/standalone.sh`
+You have two options:
+
+Option 1: Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
+
+If you are using Docker For Mac, there is an known issue. Blog post [here](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html)
```
Comment (gsmet) [inline infinispan-client/README.md]:
```suggestion
If you are using Docker For Mac, there is a known issue (see [this blog post](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html for more details).
```

You probably will drop it if you include the `hotrod-client.properties` but who knows :).

Comment (gsmet) [reply infinispan-client/README.md]: Ah so I see you included the file by default. Maybe we could drop the instructions then?

Comment (rsvoboda) [reply infinispan-client/README.md]: +1 to drop the instructions here

Comment (karesti) [reply infinispan-client/README.md]: Maybe just reword? To explain that it is not mandatory if you are not using docker 4 mac

Comment (karesti) [reply infinispan-client/README.md]: I proposed a rewording. I prefer to explain why this file is there to avoid confusion around it. WDYT?

**Pull request merged**